frontend/main.py

```diff
@@ -0,0 +1,50 @@
+"""Frontend CLI for Birdman"""
+
+import click
+
+@click.command()
+def details():
+
+    """Function for details to be written in txt file."""
+
+    venue_decided = click.confirm("Venue Decided")
+
+    if venue_decided:
+
+        meetup_link = click.prompt("Meetup Link", type=click.STRING)
+        poster_link = click.prompt("Poster Link", type=click.STRING)
+        tg_usrname = click.prompt("Telegram Username", type=click.STRING)
+        talk_decided = click.confirm("Talks Decided")
+
+        if talk_decided:
+            click.echo("Processing...(saving to .birdman/<date>/meetup.txt)\n")
+            click.echo("Join us for the next ILUG-D meetup at <Date/Time> at <Venue>\n")
+            click.echo(f"Event Page: {meetup_link}\n")
+
+            choice = click.confirm("Would you like to edit this?")
+
+            if choice:
+                details()
```

Comment:
This shouldnt redo the whole process. Instead, open the file(.birdman/<date>/meetup.txt) in nano